script.js

```diff
@@ -0,0 +1,432 @@
+            var li = "<li>" + standardArray[i] + "</li>";
+            $("#INTELLIGENCEDropList").append(li);
+        }
+    });
+    //function when the intelligence drop down is clicked
+    $('#INTELLIGENCEDropList').on("click", function (event) {
+        a = event.target.innerText;
+        $('.INTELLIGENCEbtn').text(a);
+    });
+
+    //function to fill the wisdom drop down
+    $('#WISDOMDrop').on("click", function () {
+        $("#WISDOMDropList").empty();
+        for (var i = 0; i < standardArray.length; i++) {
+            var li = "<li>" + standardArray[i] + "</li>";
+            $("#WISDOMDropList").append(li);
+        }
+    });
+    //function when the wisdom drop down is clicked
+    $('#WISDOMDropList').on("click", function (event) {
+        a = event.target.innerText;
+        $('.WISDOMbtn').text(a);
+    });
+    //function to fill the chrisma drop down
+    $('#CHARISMADrop').on("click", function () {
+        $("#CHARISMADropList").empty();
+        for (var i = 0; i < standardArray.length; i++) {
+            var li = "<li>" + standardArray[i] + "</li>";
+            $("#CHARISMADropList").append(li);
+        }
+    });
+    //function when the chrisma drop down is clicked
+    $('#CHARISMADropList').on("click", function (event) {
+        a = event.target.innerText;
+        $('.CHARISMAbtn').text(a);
+    });
+    //function when the race drop down is clicked    
+    $('#raceDrop').on("click", function (event) {
+        a = event.target.innerText;
+        $('.racebtn').text(a)
+    });
+    //function when the class drop down is clicked    
+    $('#classDrop').on("click", function (event) {
+        classCheck = event.target.innerText;
+        $('.classbtn').text(classCheck);
+        $("#spellDrop").empty();
+        fillSpell();
+    });
+    //function when the background drop down is clicked
+    $('#backDrop').on("click", function (event) {
+        a = event.target.innerText;
+        $('.backbtn').text(a)
+    });
+    //function when the level drop down is clicked    
+    $('#levelDrop').on("click", function (event) {
+        a = event.target.innerText;
+        $('.levelbtn').text(a)
+    });
+    //function to fill the class using api get method    
+    function fillClass() {
+        const pathUrl = "https://api.open5e.com/classes/";
+        $.ajax({
+            url: pathUrl,
+            type: "GET",
+            dataType: "json",
+            success: function (result) {
+                for (var i = 0; i < result.results.length; i++) {
+                    var li = "<li>" + result.results[i].name + "</li>";
+                    $("#classDrop").append(li);
+                }
+            },
+            error: function (result) {
+
+            }
+        });
+    }
+    //Function to roll a D20 using diceapi
+    $('#click-to-roll').on("click", function (event) {
+        var diceURL = "https://cors-anywhere.herokuapp.com/http://roll.diceapi.com/json/d20/"
```

Comment:
missing semicolons

this normally doesn't break anything until you run it through a minifier